src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -177,6 +214,16 @@ public Object instanceMainPostParseArgs() {
             // other exceptions with the tmp directory
             throw new UserException.BadTempDir(p, e.getMessage(), e);
         }
+        finally {
```

Comment:
I think we can move this finally block and attack it to the try where we are writing the file.